submissions/kolajoTomike/Javascript/app.js

```diff
@@ -0,0 +1,28 @@
+var productList=[
+{
+  productName:"STRAWBERRY ICE-CREAM SANDWICHES",
+  ImageUrl:"../html/img/hand.png",
```

Comment:
@tomike19 I think you should change this key name to be in `camelCase` just like the others.